db/seeds.rb

```diff
@@ -33,7 +33,7 @@
 roles = %i[trusted chatroom_beta_tester workshop_pass]
 
 num_users.times do |i|
-  name = "#{Faker::Name.name} #{Faker::Lorem.word.titleize}"
+  name = "#{Faker::Name.name} #{Faker::Lorem.word.titleize}"[0..29]
```

Comment:
The limit on name is 100 chars:

https://github.com/thepracticaldev/dev.to/blob/1dfa3d6183fdf9b281621e211848c9e090a65789/app/models/user.rb#L70

BTW `.first(N)` is more readable :D

Reply:
```suggestion
  name = Faker::Name.unique.name
```
I actually would suggest keeping it simple here, then you wouldn't need to do anything down in the email area. I was able to create 10,000 usernames.

Reply:
@maestromac we can't use unique here, see https://github.com/thepracticaldev/dev.to/pull/6058 - there's a limit in the amount of uniqueness that can be generated

Reply:
@rhymes it heavily depend on the Faker class being used. It's understandable that there's limited book title but `Faker::Name` generates from a combination of factors.
```
[3] pry(main)> 100_000.times { Faker::Name.unique.name }
=> 100000
[4] pry(main)> 10_000.times { Faker::Book.unique.title }
Faker::UniqueGenerator::RetryLimitExceeded: Retry limit exceeded for title
from /Users/macsiri/.rbenv/versions/2.6.5/lib/ruby/gems/2.6.0/gems/faker-2.10.2/lib/helpers/unique_generator.rb:30:in `method_missing'
```

I actually tried making 1mil user name but it took over 30 seconds so I stopped it 😆

Reply:
Awesome! I thought all `unique` had the same limitation, +1 on just using that!